dashboard/app/assets/stylesheets/application.scss

```diff
@@ -4099,9 +4099,56 @@ a {
   }
 }
 
+$clickable-text-glow: 0 0 1px $realyellow,
+  0 0 2px $realyellow,
+  0 0 3px $realyellow,
+  0 0 4px $realyellow,
+  0 0 5px $realyellow,
+  0 0 6px $realyellow,
+  0 0 7px $realyellow,
+  0 0 8px $realyellow,
+  0 0 9px $realyellow;
+
+@keyframes clickable-text-glow-keyframes {
+  0% {
+    text-shadow: none;
+  }
+  50% {
+    text-shadow: $clickable-text-glow;
+  }
+  100% {
+    text-shadow: none;
+  }
+}
+
 #instructions {
   .clickable-text {
     cursor: pointer;
+    position: relative;
+
+    &-with-glow {
+      animation: clickable-text-glow-keyframes 2s ease-in-out;
+
+      &:hover {
+        text-shadow: $clickable-text-glow;
+      }
+    }
+  }
+
+  .clickable-text-0 {
```

Comment:
Don't have full context here, but are we sure that there'll always be 5 clickable text, or that there won't be more than 5 clickable-texts?


nit: can be refactored like this, plus if we'll need more then 5 clickable texts - it'll be easier to  change that number
```
@for $i from 0 through 4 {
  .clickable-text-#{$i} {
    animation-delay: #{$i + 1}s;
  }
}
```

Reply:
Five was just a best guess based on existing content.  We can bump later if there is demand!  

Nice technique; now used.